tests/test.py

```diff
@@ -17,11 +17,11 @@ def test_response():
 
     assert response['status'] == 'ok'
 
-    assert len(response['predictions'][0]['embedding']) == 4
-    assert len(response['predictions'][0]['embedding'][0]) == 128
-    assert len(response['predictions'][0]['embedding'][1]) == 128
-    assert len(response['predictions'][0]['embedding'][2]) == 128
-    assert len(response['predictions'][0]['embedding'][3]) == 128
+    assert len(response['embedding']) == 4
+    assert len(response['embedding'][0]) == 128
+    assert len(response['embedding'][1]) == 128
+    assert len(response['embedding'][2]) == 128
+    assert len(response['embedding'][3]) == 128
```

Comment:
each `embedding` is a list of float?

Reply:
Yep, each embedding is a list of floats.

Eg
```
{
  "status": "ok",
  "result": [
    [
      158,
      23,
      150,
      ...
    ],
    ...,
    ...,
    [
      163,
      29,
      178,
      ...
    ]
  ]
}
```

Reply:
I guess I should update that example output while I'm at it